model/riscv_insts_zicbom.sail

```diff
@@ -0,0 +1,110 @@
+/*=======================================================================================*/
+/*  This Sail RISC-V architecture model, comprising all files and                        */
+/*  directories except where otherwise noted is subject the BSD                          */
+/*  two-clause license in the LICENSE file.                                              */
+/*                                                                                       */
+/*  SPDX-License-Identifier: BSD-2-Clause                                                */
+/*=======================================================================================*/
+
+// Cache Block Operations - Management
+//
+// Note that the CBOP prefetch instructions are existing base instructions that
```

Comment:
Why would anyone expect Zicbop instructions to be in Zicbom or Zicboz files?

Reply:
(and this is false anyway, you still want correct disassembly, and some stub Sail code)

Reply:
Good point, I removed that comment.